src/depends-on.go

```diff
@@ -0,0 +1,82 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+	"os"
+	"time"
+)
+
+func main() {
+
+	services := os.Args[1:]
+	nsb, err := ioutil.ReadFile("/var/run/secrets/kubernetes.io/serviceaccount/namespace")
+	if err != nil {
+		panic(err.Error())
```

Comment:
Maybe we should print a more descriptive error message? I saw that we just panic in many places of the code, so this comment applies to them all :)

Reply:
Actually, it prints the error. For example:
```
$ docker run meshok0/depends-on:0.1
panic: open /var/run/secrets/kubernetes.io/serviceaccount/namespace: no such file or directory

goroutine 1 [running]:
main.main()
        /go/src/depends-on/depends-on.go:35 +0xc3a
```